docs/APISocketOptions.md

```diff
@@ -124,18 +124,22 @@ their characteristics according to the following legend:
 is empty, it's an option derived from UDT. "Version 0.0.0" is the oldest version
 of SRT ever created and put into use.
 
-2. **Binding**: Defines limitation on setting the option. The field is empty if the option
+2. **Restrict**: Defines restrictions on setting the option. The field is empty if the option
 is not settable (see **Dir** column):
 
-    - `pre`: A connecting socket (both as caller and rendezvous) must be set
-prior to calling `srt_connect()` or `srt_bind()` and never changed thereafter.
-A listener socket should be set to "listening" and it will be
-derived by every socket returned by `srt_accept()`.
+	- `pre-bind`: The option cannot be altered on a socket that is already bound (by calling
+`srt_bind()` or any other function doing this, including automatic binding when trying to
+connect, as well as accepted sockets).
 
-    - `post`: This flag can be changed any time, including after the socket is
-connected (as well as on an accepted socket). Setting this flag on a listening 
-socket is effective only on that socket itself. Note though that there are some
-post-bound options that have important meaning when set prior to connecting.
+    - `pre`: Like pre-bind, but only for a connected socket (including accepted socket). If
+an option was set on a listener socket, it will be set the same on a socket returned by
+`srt_accept()`.
```

Comment:
```suggestion
- `pre`: Like `pre-bind`, but only for a connected socket (including an accepted socket). If an option was set on a listener socket, it will be set the same on a socket returned by `srt_accept()`.

  **NOTE**: The `pre-bind` characteristic applies exclusively to options that:
  - Change the behavior and functionality of the `srt_bind` call
  - Concern or set an option on the internally used UDP socket
  - Concern any kind of resource used by the multiplexer
 
 ```